config/config.schema.json

```diff
@@ -17,6 +17,53 @@
       "pattern": "^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$",
       "description": "A valid RFC1123 DNS subdomain. Multiple RFC1123 DNS label devided by a dot."
     },
+    "intOrString": {
```

Comment:
can we go with `intOrPercent` and enhance the `string` type to match a regex

Reply:
The reason I used "int or string" was because that's what K8s defines at openapi level for it so I went for that approach.

It does not go into details about the format, ranges, ... even for the rolling strategy values.

Reply:
ok. thanks for looking.